Apps/WebClient/src/featuretoggleconfig.json

```diff
@@ -63,10 +63,6 @@
         "enabled": true,
         "timelineEnabled": true,
         "datasets": [
-            {
```

Comment:
Why is this being removed.  Won't this configuration be overridden by what you put via configureSettings in your functional test?

Reply:
The settings in the featuretoggleconfig.json file is used for local only. So if it isn't removed then it will not show up when we debug the app locally. The functional test setting should override this, is my understanding.